### ISSUE TITLE
Add Brave Beta to the list of supported browsers

### ISSUE THREAD

**Issue by contribucious** [AdguardTeam/AdguardForAndroid#3401]
[Brave](https://play.google.com/store/apps/details?id=com.brave.browser_beta) browser _beta_ version was released on the Play Store on January 24, 2020.
Please add it to the list of supported browsers.

**Package name:** com.brave.browser_beta

### Thanks in advance!

---
**Related:** https://github.com/AdguardTeam/AdguardForAndroid/issues/922 (_stable_ version, not the same)
**My AG version used:** 3.3.2 (230) [stable] _(but [Nightly changelog](https://adguard.com/en/versions/android/nightly.html) checked before posting.)_

**Comment by contribucious** [comment]
Yay! Freshly [added](https://adguard.com/en/versions/android/nightly.html)!
I will close this issue therefore.
Thanks again to the team! :thumbsup:

> ## AdGuard for Android
> ### version 3.4 nightly 18 from April 21, 2020
> [...]
> [Other] Add Brave Beta to the list of supported browsers: #3401
> [...]

**Comment by contribucious** [comment]
@Revertron Feel free to add the `Resolution: Done` (or `Resolution: Fixed`) tag as needed, however. :thumbsup:

**Comment by contribucious** [comment]
@Revertron 

According to my tests, you forgot to add `com.brave.browser_beta` to `pref.enforce.https.filtering` to have HTTPS filtering active by default.

Tested OK once done manually (either by adding it manually to the `pref.enforce.https.filtering` list or by activating its option "HTTPS filtering" in the interface).

Can you confirm that? And if so, correct that? Thank you in advance. :thumbsup:

___
**Environment:** Passage from the last stable version (3.3.3) to the last beta version (3.4 beta 1) — not premium.

**Comment by contribucious** [comment]
Fixed in `3.4 beta 2`, when [Brave Nightly & Co. were added](https://github.com/AdguardTeam/AdguardForAndroid/issues/3432) (by @admitrevskiy). :thumbsup:
&nbsp;
```console
$ grep -n brave adguard_3.4_beta-1_apk_extracted/enforce_https_filtering_apps.txt
6:com.brave.browser
```
```console
$ grep -n brave adguard_3.4_beta-2_apk_extracted/enforce_https_filtering_apps.txt
6:com.brave.browser
34:com.brave.browser
35:com.brave.browser_dev
36:com.brave.browser_beta
37:com.brave.browser_default
38:com.brave.browser_nightly
```
:arrow_right: The entry for the stable version of Brave Browser is even duplicated now (see line 6 and 34). :relaxed: (cc @admitrevskiy)

**Comment by contribucious** [comment]
@artemiv4nov Hello! Small thing but as [beta 1 is coming](https://github.com/AdguardTeam/AdguardForAndroid/issues/3531) ... Thank you in advance!

_( :arrow_right: see the last sentence of the last post, concerning the double presence of `com.brave.browser` )_
